### PR TITLE
Add new runtime configuration setting description

### PR DESCRIPTION
**Pull request by akkrishnakumar**
Hello Team,

We, Runtime Core, have added a new setting to configure the runtime in `Mendix 10.0.0-beta.1`
This PR is regarding the same. 

**NOTE**: Although, we added this setting in `10.0.0-beta.1`, the spelling for this setting is different in this version. In the subsequent version, i.e. `10.0.0-beta-2`, we have changed the spelling of this setting. So I would request you to only merge this after the `Mendix 10.0.0-beta.2` release.

TIA

### CODE REVIEW

**Review by MarkvanMents — APPROVED** [review]
@akkrishnakumar 
Hi Akhil,
I've proofread this - please let me know if you have any comments.
Could I also ask you to sign the CLA. Although not strictly necessary for Mendix employees, it makes it easier to monitor our backlog.
image

Thanks.

File: content/en/docs/refguide/runtime/custom-settings/_index.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ The following custom settings can be configured:
...
 | <a id="UploadedFilesPath" href="#UploadedFilesPath">UploadedFilesPath</a> | The location of the uploaded files. A valid path can be: `\\FileServer\CustomerPortalFiles`. | [deployment folder]\data\files |
 | <a id="EnableFileDocumentCaching" href="#EnableFileDocumentCaching">EnableFileDocumentCaching</a> | Defines whether file documents should be cached. Only enable this if you are sure that the file documents will not contain sensitive information. Images are always cached. | false |
+| <a id="StrictChangeBehavior" href="#StrictChangeBehavior">StrictChangeBehavior</a> | Defines the behavior when changing values of Enums and Calculated attributes.<br/>When set to true, setting an invalid value for an Enum attribute and/or setting a value for a Calculated attribute will result in an InvalidEnumerationValueException and/or ReadOnlyAttributeException respectively.<br/>When set to false, changes to the values of Enums and/or Calculated attributes will be allowed.<br/>We plan to remove this setting in Mendix version 11, after which, an exception will always be raised when setting an invalid value. | true |
```
Comment (atennapel) [inline content/en/docs/refguide/runtime/custom-settings/_index.md]: I think the setting is now `ObjectManagement.StrictChangeBehavior`.

**Comment by atennapel** [comment]
Additionally we need to add the release note for this new setting to the beta1 release notes. Were you planning to do that in another MR @akkrishnakumar ? You can also consider adding it to this one.

**Comment by akkrishnakumar** [comment]
> Additionally we need to add the release note for this new setting to the beta1 release notes. Were you planning to do that in another MR @akkrishnakumar ? You can also consider adding it to this one.

I think that has been covered in this [commit](https://github.com/mendix/docs/commit/8af27583cb70463de1992a5ba9b7bb8501107edb). This has the old name. When  `beta 2` is released, it will pickup up the new release note with the new name.

**Comment by atennapel** [comment]
Great! Then approved from my side.

**Comment by akkrishnakumar** [comment]
@atennapel Could also resolve your comment, if it has been resolved correctly ?

**Review by atennapel — APPROVED** [review]
Approved.

**Pull request merged**